nss-day-cohort-41/tabloid-fullstack-shocking-secret-keepers
# Sf initial

# Description
This pull request refers to issues in the following tickets:
Ticket #19 
Ticket #20 
1. User should be able to view comments for a specific post (still need to add 'view comments' button in post details page when able so can dynamically route to that page and use the params id to get all comments for that specific post)
2. User should be able to add a new comment to a post (need to add "add comments" button in post details for dynamic routing)
3. Once user clicks 'save comment' they should be routed to the comments list for that individual post and see the newly added comment as most recent one on list
4. Currently 'back to post' button does not route anywhere, but will route back to specific details page for post when able
5. Currently "edit" and "delete" buttons are not functioning on individual comments

## Type of change

# Testing Instructions
1. fetch branch sf-initial
2. run Tabloid app
3. npm start in terminal
4. View comments for specific post by typing in url: http://localhost:3000/commentsbypost/1 (you should see all the posts for postId = 1) 
5. Add comments for that specific post by typing in url: http://localhost:3000/comments/add/1 (you should be routed to new comments form for postId =1)
6. After clicking save comment you should be directed to comments page for that specific post

# Checklist:
